osu.Game/Screens/Play/SkipButton.cs

```diff
@@ -182,6 +182,7 @@ private class Button : OsuClickableContainer
             private FillFlowContainer flow;
             private Box background;
             private AspectContainer aspect;
+            private bool isClicked;

```

Comment:
I probably would disable the button via the `Enabled` variable instead of creating a new one.